README.md

```diff
@@ -13,6 +13,16 @@ Important notes:
 
 ### Using command line
 
+#### Install Dependencies
+
+- Python - We recommend you use the system version where possibly.
+    - Version 2.7
+- [NumPy](http://www.numpy.org/) - Can be installed through some system package managers or via [pip](https://pip.pypa.io/)
+    - Version 1.9.2
+- [CMake](http://www.cmake.org/)
+- [Cap'n Proto](https://capnproto.org/)
+    - Version 0.5.3
```

Comment:
is the version correct? Because the one I've reinstalled today was 0.5.2 and that seems to fix my problem. (pycapnp is 0.5.5, right?)